shared/js/background/declarative-net-request.js

```diff
@@ -327,6 +331,16 @@ export async function refreshUserAllowlistRules (allowlistedDomains) {
     await updateUserAllowlistRule(normalizedAllowlistedDomains)
 }
 
+export async function flushSessionRules () {
+    chrome.declarativeNetRequest.getSessionRules().then(rules => {
+        if(rules.length) {
+            console.log('removing session rules', ruleIds)
+            const ruleIds = rules.filter(r => isValidSessionId(r.id)).map(r => r.id)
+            return chrome.declarativeNetRequest.updateSessionRules({ removeRuleIds: ruleIds})
```

Comment:
Nit: Might as well `await` here too, so that the returned Promise only resolves when this is done. So far, it will resolve before then which could be a foot-gun in the future.